tracim/tracim/model/auth.py

```diff
@@ -159,7 +159,7 @@ def calendar_url(self) -> str:
             domain=cfg.RADICALE_CLIENT_HOST or request.domain,
             port=cfg.RADICALE_CLIENT_PORT,
             id=self.user_id,
-            slug=slugify(self.get_display_name(
+            extra='#' + slugify(self.get_display_name(
```

Comment:
rename "extra" to "anchor-extra" et mettre le # comme il était auparavant (except if the regular expression is compatible with anything after the .ics sequence)